tests/data/excelTestFile_services_expected.json

```diff
@@ -53,9 +53,9 @@
       }
     }
   ],
-  "synchronisation": [
+  "synchronization": [
```

Comment:
We should document a spelling guide so that we are consistent w/r/t American-vs-UK spellings.

(I don't care which we prefer.)

Reply:
Sure!